astropy/coordinates/angle_utilities.py

```diff
@@ -216,8 +213,6 @@ def uniform_spherical_random_volume(size=1, max_radius=1):
         The number of points to generate.
     max_radius : number, quantity-like, optional
         A dimensionless or unit-ful factor to scale the random distances.
-    rng : `numpy.random.Generator`, optional
```

Comment:
While it's correct that this function doesn't accept an `rng` argument, I think one should be added!
Definitely as a followup, so it's right to remove it here.

Reply:
Maybe @adrn can explain. He added this in #11628 .

Reply:
Ah, that docstring is vestigial ... I had originally added the rng option but see discussion here: #11628 and #11687

Reply:
Sounds a worthwhile Astropy-wide goal.

Reply:
(I have a branch where I started down this path -- I'll see what state it's in!)